openedx/features/wikimedia_features/views.py

```diff
@@ -0,0 +1,15 @@
+from django.shortcuts import render
+
+# Create your views here.
+
+import json
+from django.http import HttpResponse
+
+def dummy(request):
```

Comment:
Can we please remove these dummy views ?

Reply:
sure